src/tests/Main/ComparisonTests.fs

```diff
@@ -141,6 +142,24 @@ let ``Record equality works``() =
     Object.ReferenceEquals(r1, r1) |> equal true
     Object.ReferenceEquals(r1, r2) |> equal false
 
+
+[<Test>]
+let ``Record equality ignores dynamic fields``() =
+    let r1 = { a = 1; b = 2 }
+    r1?c <- 1
+    let r2 = { a = 1; b = 2 }
+    r2?c <- 2
+    equal true (r1 = r2)
+    equal false (r1 <> r2)
+
+[<Test>]
+let ``Record comparison ignores dynamic fields``() =
+    let r1 = { a = 1; b = 2 }
+    r1?c <- 1
+    let r2 = { a = 1; b = 2 }
+    r2?c <- 2
+    equal 0 (compare r1 r2)
+
```

Comment:
Oh, thanks for the tip! Was under impression that ``.\build`` includes ``NUnitTests`` step by default. Fixed.